src/playwright-api/makeTest.ts

```diff
@@ -38,7 +38,7 @@ export const makeTest = (
         const completeArchive = await createResourceArchive(page);
         await use();
 
-        await takeArchive(page, testInfo);
+        const sourceMap: Map<string, string> = await takeArchive(page, testInfo);
```

Comment:
We won't need the type here since `takeArchive` has it specified as the return type.